### PR TITLE
Likes and single video page

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks please review my PR for Likes Context and Video listing page.

Screenshot 2022-05-15 014828

Screenshot 2022-05-15 015453

### CODE REVIEW

File: src/components/likedvideospage/LikedVideosPage.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { useLikes } from "../../context";
+import VideoCard from "../videocard/VideoCard";
+import "./likedvideopage.css"
+
+export default function LikedVideosPage() {
+
+    const {likedVideos} = useLikes()
+
+    return <div className="grand-main">
+        <h2 className="p-y-1">Liked Videos:</h2>
```
Comment (varun13169) [inline src/components/likedvideospage/LikedVideosPage.jsx:11]: What is the naming convention for css class being followed ?

h2 className="p-y-1"???

File: src/context/index.js [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 import AuthProvider from "./AuthProvider";
 import VideoProvider from "./VideoProvider";
+import LikeProvider from "./LikeProvider";
```
Comment (varun13169) [inline src/context/index.js:5]: create an index.js for all the providers and then import from that index.js

File: src/components/likedvideospage/LikedVideosPage.jsx (hunk 1 above)
Comment (bhushannaikaroskar) [reply src/components/likedvideospage/LikedVideosPage.jsx:11]: That's an utility class for padding

File: src/context/index.js (hunk 2 above)
Comment (bhushannaikaroskar) [reply src/context/index.js:5]: Didn't get you, This is same index file you are talking about I guess. I have imported all the contexts and then using it from same index file

**Review by varun13169 — APPROVED** [review]

**Review by iRohitGaur — APPROVED** [review]
Nice work! 

Good to merge 👍🏻

**Review by srishtigautam21 — APPROVED** [review]
The code looks good
Good to merge

**Pull request merged**